ga4gh/serverconfig.py

```diff
@@ -45,6 +58,38 @@ class ProductionConfig(BaseConfig):
     DATA_SOURCE = None
 
 
+class GoogleOIDCConfig(ProductionConfig):
```

Comment:
[My solution](https://github.com/Genecloud/simplebeacon/blob/master/beacon_rp.py#L153), which isn't any better, but at least fits on one page.  Like I mentioned above, I'm assuming that I'm always configuring from JSON documents that we've seen wild OP implementations produce.  That assumption might not hold, but I think there's some wisdom for OPs to provide this and it will ultimately be commonplace.